spec/requests/api/subscriptions/update_spec.rb

```diff
@@ -0,0 +1,38 @@
+RSpec.describe 'POST /api/subscriptions', type: :request do
+  let(:user) { create(:user) }
+  let(:credentials) { user.create_new_auth_token }
+
+  subject { response }
+
+  describe 'as a authenticated user, when subscribing' do
+    before do
+      post '/api/subscriptions', params: { user: { id: user.id } }, headers: credentials
```

Comment:
Do you really need to pass i n the user id in params? The request contains the user info in headers.

Reply:
Indeed we do not, it's been removed